src/utils/index.js

```diff
@@ -1,8 +1,7 @@
-import ApiError from './ApiError';
-import UserResponse from './UserResponse';
-import Helpers from './helpers';
-import Mailer from './mailer';
+import ApiError from './apiError';
```

Comment:
It will be nice to change ApiError too to camelCase
```suggestion
import apiError from './apiError';
```

Reply:
Yes @emmanuelnwankwo I wanted to however thats the only class that gets called with the new keyword because if extends from the javascript error class